content/SCALE/SCALETutorials/Dashboard/_index.md

```diff
@@ -1,19 +1,82 @@
 geekdocCollapseSection: true
 weight: 4
 tags:
 - dashboard
 related: false
 ---
 
-This section contains tutorials involving the [SCALE Dashboard]({{< relref "SCALEDashboard.md" >}}).
+This section contains tutorials for the [main Dashboard]({{< relref "SCALEDashboard.md" >}}).
+
+## Customizing the Dashboard
+You can customize the main **Dashboard** by moving, adding, or deleting widgets.
+
+Click **Configure** to put the **Dashboard** into configuration mode.
+
+While in configuration mode all widgets show as widget groups that are surronded by dotted line boxes. Each widget group includes a drag handle, and the edit and delete icon buttons.
+
+### Moving a Widget
+To move a widget to a new position on the **Dashboard** screen, click **Configure** to put the screen into configuration mode.
+
+Locate the widget you want to reposition.
+
+Click on and hold the drag handle at the top center of the widget group, then drag the widget to the desired position on the screen.
+The moved widget shifts the widget at the screen position one position to either the right or left based on the position on the **Dashboard** screen.
+
+After moving widgets, click **Save** at the top right of the screen to exist configuration mode and show widgets in the new positions.
+
+### Adding or Editing a Widget
+You can add new widgets to the dashboard or change existing widgets to a new layout, category, or type.
+
+1. Click **Configure** at the top right of the screen to put the screen into configuration mode.
+   
+   If adding a new widget, click **Add** to open the **Widget Editor** with no existing layout or other settings.
+
+   If changing an existing widget, locate the widget group on the screen, then click **Edit** at the top right of that widget group to open the **Widget Editor** with the layout and settings for that widget group.
+
+2. Click on the layout image you want to use. The image on the screen show the new widget layout.
+   
+   If adding a new widget, the default layout is full size with the category and type set to **Empty**.
+
+   If editing an existing widget, the current layout changes to show the existing category and type in the first widget of the new layout.
+   An error shows in the selected widget of the group if the widget size does not support the selected category and type.
+
+3. Select the widget in the group you want to add or change.
+   If the layout includes half and/or quarter size widgets, the first widget in the group is selected by default.
+
+   If not configuring the default widget, select the widget size (half or quarter size) and postion in the group you want to configure.
+
+4. Select the **Widget Category** and **Widget Type** to apply to the selected widget.
+   For example, if configuring a network widget, you can use one full size layout or select one with half and quarter size widgets.
```

Comment:
Isn't this more about the layout than the Widget Category and Type? It seems like the proceedure should be
2. Select the layout (for example, half and quarter)
3. If more than one widget in the layout, select the widget you want to edit
4. Select Category and Type (for example, network OR options are apps, network, etc.)

Reply:
no, this is about setting up the selected widget with the category and type. you need to do this for each widget in the group.